tests/main/debs/task.yaml

```diff
@@ -1,12 +1,16 @@
-summary: Ensure that our debs have the "built-using" header
+summary: Ensure our debs are correctly built
 
 systems: [-ubuntu-core-*, -fedora-*, -opensuse-*, -arch-*, -amazon-*, -centos-*]
 
 execute: |
+    echo "Ensure that our debs have the 'built-using' header"
     out=$(dpkg -I "$GOHOME"/snapd_*.deb)
     if [[ "$SPREAD_SYSTEM" = ubuntu-* ]]; then
         # Apparmor & seccomp is only compiled in on Ubuntu for now.
         echo "$out" | MATCH 'Built-Using:.*apparmor \(='
         echo "$out" | MATCH 'Built-Using:.*libseccomp \(='
     fi
     echo "$out" | MATCH 'Built-Using:.*libcap2 \(='
+
+    echo "Ensure that the snapd.session-agent.socket symlinks is part of the deb and that it has the right (relative) target"
+    dpkg -c "$GOHOME"/snapd_*.deb |grep -- '-> \.\./snapd.session-agent.socket'
```

Comment:
There aren't that many files in the package, so we could use a `MATCH -- ..` here.

Fails on 14.04:
```
  .
  Start with '\''snap list'\'' to see installed snaps.'
+ echo 'Ensure that the snapd.session-agent.socket symlinks is part of the deb and that it has the right (relative) target'
Ensure that the snapd.session-agent.socket symlinks is part of the deb and that it has the right (relative) target
+ grep -- '-> \.\./snapd.session-agent.socket'
+ dpkg -c /home/gopath/snapd_1337.2.40~14.04_amd64.deb
-----
```